regression/esbmc-cpp/OM_sanity_checks/definitions/main.cpp

```diff
@@ -0,0 +1,4 @@
+#include "definitions.h"
```

Comment:
I don't understand what this file is testing. On a regular C++ implementation, this should fail to compile because `"definitions.h"` is not in the include-path.

Reply:
This test case includes `definitions.h` in the same way as other C++ OMs, e.g.
https://github.com/esbmc/esbmc/blob/master/src/cpp/library/algorithm
https://github.com/esbmc/esbmc/blob/master/src/cpp/library/cmath

`definitions.h` is in `-I <path>/src/cpp/library`.

Reply:
Sure, but `"definitions.h"` is an implementation detail and the non-existence of it should not constitute a regression. If anything, it should not exist (without a prefix like `__ESBMC` or similar). Anyhow, since it's merged we can discuss files like this when cleaning up the ESBMC-provided C++ headers.